package.json

```diff
@@ -84,6 +84,7 @@
     "@fortawesome/react-fontawesome": "^0.1.3",
     "@portis/web3": "^2.0.0-beta.43",
     "@sentry/browser": "^5.0.8",
+    "@toruslabs/torus-embed": "^1.2.0",
```

Comment:
please bump to 1.2.1

Reply:
Bumping to 1.2.3, the latest.

Reply:
Also grabbing the latest web3modal, no long in beta.